WorkloadManagementSystem/Service/PilotsLoggingHandler.py

```diff
@@ -0,0 +1,85 @@
+""" PilotsLoggingHandler is the implementation of the PilotsLogging service
+
+    The following methods are available in the Service interface
+
+    addPilotsLogging()
+    getPilotsLogging
+    setPilotsUUIDtoIDMapping()
+    addPilotsUUID()
+    deletePilotsLogging()
+
+"""
+
+__RCSID__ = "$Id: $"
+
+from DIRAC.WorkloadManagementSystem.DB.PilotsLoggingDB import PilotsLoggingDB
+from DIRAC.Resources.MessageQueue.MQCommunication import createConsumer
+
+from DIRAC.Core.DISET.RequestHandler import RequestHandler
+from DIRAC import S_OK
+
+
+class PilotsLoggingHandler( RequestHandler ):
+  """Server side functions for Pilots Logging service"""
+
+  @classmethod
+  def initializeHandler( cls, serviceInfo ):
+    """Initialization of Pilots Logging service
+    """
+    cls.consumersSet = set()
+    cls.pilotsLoggingDB = PilotsLoggingDB()
+
+    result = createConsumer( "lbvobox50.cern.ch::Queue::test", callback = cls.consumingCallback ) # XXX hardcoded URI
+    if result['OK']:
+      cls.consumersSet.add(result)
```

Comment:
and if not `'OK'`?

Reply:
That's a very good question. What should we do? React or just look the other way? I'm afraid it's pretty easy to fall into absurdity, when there are logging issues...
![image](https://user-images.githubusercontent.com/1897515/31376922-bd234b5a-ada6-11e7-90e0-005da1ab61e5.png)
;)

Reply:
But this is for creating a consumer, no?

Reply:
You're right. It's fixed now.